run-make-check.sh

```diff
@@ -59,7 +59,7 @@ function main() {
         cmake_opts+=" -DWITH_SEASTAR=ON"
     fi
     configure $cmake_opts $@
-    build tests && echo "make check: successful run on $(git rev-parse HEAD)"
+    build tests && echo "make check: successful build on $(git rev-parse HEAD)"
```

Comment:
```suggestion
    build tests && echo "make check: successful built on $(git rev-parse HEAD)"
```

Reply:
This wouldn't be grammatically correct -- "build" is a noun here, not a verb.

Reply:
ahh, right! otherwise it'd be "successfully built" then.